build/ci-tf-bin.Dockerfile

```
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS builder
```

```suggestion
FROM registry.access.redhat.com/ubi8/go-toolset:1.19 AS builder
```
it can save us the golang installation and also set us to a specific version (instead of being unpinned)

---

go-toolset:1.19 is 1.21GB and ubi-minimal is 94.5MB. Installed size of go-toolset package and its dependencies is 593MB so I don't know how important is to set the specific golang version vs double the image size.

---

Ack. If it's that much more space then I agree with you. Let's keep using the original image